src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -319,6 +326,22 @@
             },
             false
         );
+        availableField.addEventListener(
+            'click',
+            (event) => {
+                currentDraggedItem = event.currentTarget;
+                sourceContainer = currentDraggedItem.parentNode;
+                draggedItemPosition = -1;
+                targetContainer = doc.querySelector(
+                    '.ibexa-collapse--field-definitions-group.ibexa-collapse--active-field-definitions-group .ibexa-content-type-edit__field-definition-drop-zone'
+                );
+
+                if (targetContainer) {
```

Comment:
nitpick: probably we can add this check at the beginning and then proceed with preparing data for `addField`.